checker/server.go

```diff
@@ -53,8 +58,10 @@ func (as *authServer) Check(ctx context.Context, req *authz.CheckRequest) (*auth
 	store.Read(func(ps *policystore.PolicyStore) { st = checkStore(ps, req) })
 	resp.Status = &st
 	log.WithFields(log.Fields{
-		"Request":  req,
-		"Response": resp,
+		"Req.Method":   req.GetAttributes().GetRequest().GetHttp().GetMethod(),
+		"Req.Path":     req.GetAttributes().GetRequest().GetHttp().GetPath(),
+		"Req.Protocol": req.GetAttributes().GetRequest().GetHttp().GetProtocol(),
+		"Response":     resp,
```

Comment:
not your bug, but I think this "check complete" log should also be at Debug level.  At Info level it means we are writing logs for every request, even if debug logging is turned off, which might hurt performance.